src/core/prototypes/state.ts

```diff
@@ -153,12 +153,17 @@ export class ChartStateManager {
       for (let index = 0; index < origin.constraints.length; index++) {
         const originConstringts = origin.constraints[index];
         const current = chart.constraints[index];
-        expect_deep_approximately_equals(
-          originConstringts,
-          current,
-          defaultDifferenceApproximation,
-          true
-        );
+        try {
+          expect_deep_approximately_equals(
```

Comment:
using throw/catch for deep equality check is not advised, but fixing this is outside the scope of this bug fix

Reply:
I will rework in next time